packages/@react-spectrum/actionbar/src/ActionBar.tsx

```diff
@@ -125,7 +125,9 @@ const ActionBarInner = React.forwardRef((props: ActionBarInnerProps, ref: DOMRef
       </div>
     </FocusScope>
   );
-});
+}
+
+const ActionBarInnerWithRef = React.forwardRef(ActionBarInner) as <T>(props: SpectrumActionBarProps<T> & {ref?: DOMRef<HTMLDivElement>}) => ReturnType<typeof ActionBarInner>;
```

Comment:
whyyyyy?

Reply:
because of typescript :( this was the best interpretation of this discussion I could come up with https://stackoverflow.com/questions/58469229/react-with-typescript-generics-while-using-react-forwardref